charts/spark-operator-chart/templates/webhook-init-job.yaml

```diff
@@ -12,6 +12,9 @@ spec:
     metadata:
       name: {{ include "spark-operator.fullname" . }}-webhook-init
       {{- if .Values.istio.enabled }}
+      annotations:
+        "sidecar.istio.io/inject": {{ .Values.istio.enabled | quote }}
+      {{- else }}
       annotations:
         "sidecar.istio.io/inject": "false"
       {{- end }}
```

Comment:
I mean:
```suggestion
      annotations:
        "sidecar.istio.io/inject": {{ .Values.istio.enabled | quote }}
```

Reply:
I thought you might have meant this. I haven't done much helm in awhile. If `istio.enabled` isn't defined, would this cause an error?

Provided the answer is that there won't be an error, I'm totally happy with this :)

Reply:
In this case, you can use `{{ .Values.istio.enabled | default false | quote }}`
So `null` or `""` (empty string) will be `"false"`

Reply:
https://github.com/GoogleCloudPlatform/spark-on-k8s-operator/pull/1439/commits/6e507f651cbe66f688bf9df9d62fa7d674c6ae6f

Reply:
So I wonder if the default here is supposed to be false.

Its come to my attention that istio prevents jobs from exiting because the sidecar container doesn't exit. Which means that the original intent of the code would be correct.

Thinking about it, I think the solution is actually leaving this code as is, and guiding the users to add pod labels that prevent istio from adding the sidecar.

I will admit that this solution would require some documentation as it's not immediately obvious how to proceed.